entries/k/kahoot.it.json

```
@@ -0,0 +1,12 @@
+{
+  "Kahoot!": {
+    "domain": "kahoot.it",

```

```suggestion
    "domain": "kahoot.com",
    "additional-domains": [
      "kahoot.it"
    ],
```
While the quizzes are accessible from `kahoot.it`, the account login, management and other sensitive details (e..g private kahoots) are available at `kahoot.com`. Please also change the file name to `kahoot.com.json`.

I've also suggested adding `kahoot.it` as an additional domain, as there are some login pages that use subdomains of that (e.g. `create.kahoot.it`).